Makefile

```diff
@@ -1,6 +1,6 @@
 SHELL := /bin/bash
 
-PYTEST_ARGS=
+PYTEST_ARGS= --rich-gen
```

Comment:
Why the flag is always on? OMHO it can modify generated files during the test run on CI without letting us know.

Reply:
I just forgot to remove it, sorry.